internal/c/libqb/src/mac-mouse-support.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#ifdef QB64_MACOSX
```

Comment:
It's a small thing in this case, but rather than require the caller to check `QB64_MACOSX` every time they want to call these functions I think it's better to declare stubs in here that don't do anything on the other OSs. That way callers don't have to check and we don't need to add `#ifdef`s everywhere. IE:

```c
#ifdef QB64_MACOSX
void macMouseInit();
void macMouseDone();
#else
static inline void macMouseInit() { }
static inline void macMouseDone() { }
#endif
```

Reply:
Yeah. This looks a lot cleaner. Done.